Source/Applications/TsfManager/App.xaml.cs

```diff
@@ -98,6 +101,12 @@ public Guid NodeID
 
         #region [ Methods ]
 
+        private void Application_Startup(object sender, StartupEventArgs e)
```

Comment:
I think this is just the test app in GSF, so I presume if these are "accepted" changes, the next step would be distribution of this code to openPDC, openHistorian, SIEGate and ProjectAlpha at a minimum?

Reply:
You answered this question in PR notes! :-)

Reply:
Yes. I was also thinking of substationSBG and openMIC.